sa3ed_app/www/lost_person_create_detail/index.js

```diff
@@ -1,6 +1,20 @@
 var app = Vue.createApp({
     delimiters: ['[[', ']]'], // Change delimiters here
     data() {
+        frappe.call({
+            method:"sa3ed_app.api.WhitelistBypass.get_country_list",
+            callback: function(result) { 
```

Comment:
In callbacks you should apply in kind of validation on the result to make sure it's valid for what you are going to do with the response in your function.

This should apply here and where applicable.

Reply:
This issue should be solved by Commit 8dc005d